src/style.css

```diff
@@ -1,3 +1,96 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: gray;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  max-width: auto;
+  max-height: auto;
+  margin: auto;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  position: relative;
+  display: block;
+  width: auto;
+  height: auto;
+  margin: auto;
+}
+
+.nav__link {
+  color: black;
+  text-decoration: none;
+  display: block;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  justify-content: flex-end;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.link1 {
+  position: relative;
+  color: #00acdc;
+}
+
+.link2:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  display: flex;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  margin: 0 10px;
```

Comment:
Be consistent with your vertical margins. It's better to add only top or only bottom margin, but not both, to avoid collapsing margins in some cases.